xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp

```diff
@@ -170,6 +170,9 @@ void CGUIDialogVideoManagerVersions::SetDefaultVideoVersion(const CFileItem& ver
   // set the specified video version as default
   m_database.SetDefaultVideoVersion(itemType, dbId, version.GetVideoInfoTag()->m_iDbId);
 
+  // update default artwork
+  m_database.SetArtForItem(dbId, m_videoAsset->GetVideoInfoTag()->m_type, version.GetArt());
```

Comment:
In which situation does this help? For me it breaks the case where you add a movie B with art as a new version of movie A that already has art, then make the version of B the default. Exit the library, return to the library: Chose version to play/versions as folder/Manage versions show both versions with movie B art. It may be important that movie A existed in the library before video versions, and as a result doesn't have art with media_type 'videoversion'.

Reply:
@ksooo have an answer for this point?